packages/@webex/plugin-meetings/src/multistream/remoteMedia.ts

```diff
@@ -89,6 +90,13 @@ export class RemoteMedia extends EventsScope {
 
   public readonly id: RemoteMediaId;
 
+  /**
+   * The max frame size of the media request, used for logging and media requests.
+   * Set by setSizeHint() based on video element dimensions.
+   * When > 0, this value takes precedence over options.resolution in sendMediaRequest().
+   */
+  public maxFrameSize = 0;
```

Comment:
I don't think it should be public, because we normally want the app to set it via setSizeHint(). If someone sets it directly, things won't work, because `this.receiveSlot?.setMaxFs()` won't be called.